pkg/controller/node/ipam/cloud_cidr_allocator.go

```diff
@@ -207,20 +210,19 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 	}
 	podCIDR := cidr.String()
 
-	for rep := 0; rep < cidrUpdateRetries; rep++ {
-		node, err = ca.nodeLister.Get(nodeName)
-		if err != nil {
-			glog.Errorf("Failed while getting node %v to retry updating Node.Spec.PodCIDR: %v", nodeName, err)
-			continue
-		}
+	node, err = ca.nodeLister.Get(nodeName)
```

Comment:
I don't think this is what we want. In general, retries are protecting us from conflicts - back-to-back retry should actually solve the problem of conflict.

Can you describe the the problem you are trying to solve with this change?